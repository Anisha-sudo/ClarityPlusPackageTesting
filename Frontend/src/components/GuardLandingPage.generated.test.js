import renderer from 'react-test-renderer';
import React from "react";
import GuardDashboard from './GuardDashboard';
import imgPath from "./Images/college.jpeg"
import GuardLandingPage from "./GuardLandingPage";

jest.mock("@mui/material");
jest.mock('./GuardDashboard');
jest.mock("./Images/college.jpeg");

const renderTree = tree => renderer.create(tree);
describe('<GuardLandingPage>', () => {
  it('should render component', () => {
    expect(renderTree(<GuardLandingPage 
    />).toJSON()).toMatchSnapshot();
  });
  
});