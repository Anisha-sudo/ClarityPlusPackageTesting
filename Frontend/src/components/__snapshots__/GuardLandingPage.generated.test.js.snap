// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<GuardLandingPage> should render component 1`] = `
<img
  src="college.jpeg"
  style={
    Object {
      "display": "block",
      "margin": "auto",
      "textAlign": "center",
      "verticalAlign": "middle",
      "width": "100%",
    }
  }
/>
`;
